### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by adnanjpg**
#### Docs

- Updated graphql_flutter/README.md

### CODE REVIEW

File: packages/graphql_flutter/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ First, depend on this package:
 
 ```yaml
 dependencies:
-  graphql_flutter: ^4.0.0-beta
+  graphql_flutter: ^5.0.0
```
Comment (vincenzopalazzo) [inline packages/graphql_flutter/README.md:54]:
```suggestion
  graphql_flutter: ^<last version on github release>
```

File: packages/graphql_flutter/README.md [hunk 2]
```diff
@@ -61,14 +61,14 @@ import 'package:graphql_flutter/graphql_flutter.dart';
 ```
 
 ## Migration Guide
-
+<!-- TODO: initialize changelog-v4-v5 file and update the link here -->
```
Comment (vincenzopalazzo) [inline packages/graphql_flutter/README.md:64]: can you remove this HTML line from here? for now it is not a TODO for the team
```suggestion
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Thanks for updating the code in the readme, I was thinking to remove it but for now, I have comment unrelated to the code

**Review by adnanjpg — COMMENTED** [review]
Hi @vincenzopalazzo  , I've accepted your suggestions. please check it out

**Comment by adnanjpg** [comment]
I think I've accepted your suggestions but I'm not sure it worked, Sorry I'm new to this

**Comment by vincenzopalazzo** [comment]
>I think I've accepted your suggestions but I'm not sure it worked, Sorry I'm new to this

you need to change your code, and make another commit after that you change are reflected on the PR

**Review by vincenzopalazzo — APPROVED** [review]
Thanks to make this work :)

Hope to see more contribution on this repository

**Pull request merged**